<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1221</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Choose Output Directory and Images</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Image Directory:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="image_directory">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>100</y>
        <width>651</width>
        <height>41</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTreeWidget" name="picture_dir_tree">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>21</y>
        <width>501</width>
        <height>531</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>35</width>
        <height>35</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">Choose one or multiple image files for calibration tests:</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="output_directory">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>260</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>190</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Data Directory:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="change_image_directory">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>170</y>
        <width>181</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Directory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="change_output_directory">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>330</y>
        <width>181</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Directory</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="use_i607">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>410</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Use i607 images</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="use_i203">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>460</y>
        <width>231</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Use i203 images</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>340</y>
        <width>341</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Image Sizes for Computations:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_selected_images">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>420</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert Selected Images</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Measure Angle</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1221</width>
        <height>581</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>K1 - Gradient Algorithm</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>K2 - Rotatation Algorithm</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>MO - Radial Histogram Algorithm</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="log_text">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>1191</width>
      <height>251</height>
     </rect>
    </property>
    <property name="plainText">
     <string>&gt; Choose images to work with
&gt; Press Convert Selected Images to output them and leave them ready.
&gt; Choose angle compuation algorithm and run it.

Note that if &quot;Convert Selected Images&quot; is not clicked before running the algorithms, this will be done internally anyway.
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
